nix/devshell.nix

```diff
@@ -19,6 +20,7 @@
         pkg-config
         python3
         python3Packages.pip
+        self'.packages.kernel-hardening-checker
```

Comment:
One other thing that could benefit from improvement here would be to generalize the `devShells.kernel-[x86-64|aarch64|riscv64]` so that depending on the HW architecture, the `inputsFrom = [pkgs.linux_latest|jetpack-nixos.<right_kernel_package_here>|<riscv64-referencedevicekernelpackagehere>]`